dirty_cat/minhash_encoder.py

```diff
@@ -73,6 +73,7 @@ def __init__(self, n_components=30, ngram_range=(2, 4),
         self.minmax_hash = minmax_hash
         self.count = 0
         self.handle_missing = handle_missing
+        self.capacity = 2**10
```

Comment:
Given that this parameter is not in the init and not documented, I wonder whether we should not make it private: "_capacity". WDYT?

Reply:
That makes sense. Changed it to _capacity.